blocks/api/registration.js

```diff
@@ -68,6 +68,12 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( 'focusable' in settings && ! ( isFunction( settings.focusable ) || isBoolean( settings.focusable ) ) ) {
```

Comment:
Note that I still need to figure out how to nicely handle selection when tabbing into a block that displays things when the focus property is set.
For example the "button" block has 3 focusable elements:
- The button text
- A link (only shown when the block is focused)
- A button associated with the link (only shown when the block is focused)
This gives inconsistent results when tabbing backwards because the link and link-button are skipped over to select the button-text but they aren't when tabbing forwards because they are displayed once the button text has been selected.